src/ui/components/ViewError/ViewError.tsx

```diff
@@ -131,7 +133,7 @@ export function ViewError({
               if (pageTemplateType !== 'dialog') {
                 navigate('/');
               }
-              window.location.reload();
```

Comment:
Can something break because we're no longer reloading the page?

Reply:
Good question. I'll check

Reply:
I don't know, tbh.
My logic was: we are updating a prop for ErrorBoundary, so it should be updated after going to home screen. However, I think window.reload can be helpful

Reply:
If we're reloading, what does `onRetry` accomplish?

Reply:
This looked like GlobalBoundary wasn't updated during this reload properly. I will check again